lib/ApplicationFeatures/VersionFeature.h

```diff
@@ -38,6 +38,7 @@ class VersionFeature final : public application_features::ApplicationFeature {
 
  private:
   bool _printVersion;
+  bool _printVersionJson;
```

Comment:
This variable is nowhere initialized, so it may be `true` by chance, which will prevent a server start :scream: